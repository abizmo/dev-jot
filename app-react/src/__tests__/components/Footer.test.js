import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

describe('Footer tests', () => {
  test('should render and match snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('a').first().text()).toBe('Dighital');
  });
});
