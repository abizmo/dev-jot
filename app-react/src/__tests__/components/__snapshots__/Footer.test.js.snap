// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests should render and match snapshot 1`] = `
<styled.footer
  className="footer"
>
  <styled.div>
    Icons made by  
    <a
      href="https://www.flaticon.com/authors/dighital"
      title="Dighital"
    >
      Dighital
    </a>
      from  
    <a
      href="https://www.flaticon.com/"
      title="Flaticon"
    >
      www.flaticon.com
    </a>
  </styled.div>
</styled.footer>
`;
